lib/commons/color/get-foreground-color.js

```diff
@@ -63,6 +64,11 @@ function getForegroundColor(node, _, bgColor) {
     return null;
   }
 
+  if (fgColor.alpha === 0) {
```

Comment:
I think we should probably blend text-shadow any time the alpha is < 1.

Reply:
We're currently blending the text-shadow into the background color most of the time if it's a thin shadow. Should we be blending it with the foreground color too, or only doing one or the other?

Reply:
It can effect both, so I think it should be both.